package.json

```diff
@@ -20,34 +21,34 @@
     "build": "gulp build",
     "lint": "resin-lint lib tests gulpfile.coffee",
     "test-node": "gulp test",
-    "test-browser": "karma start",
+    "test-browser": "mockttp -c karma start",
     "test": "npm run lint && npm run test-node && npm run test-browser",
-    "prepublish": "npm test && npm run build"
+    "prepack": "npm run build"
   },
   "author": "Juan Cruz Viotti <juan@resin.io>",
   "license": "Apache-2.0",
   "devDependencies": {
+    "balena-config-karma": "^2.3.1",
+    "balena-request": "^10.0.0",
     "chai": "^3.5.0",
     "chai-as-promised": "^6.0.0",
-    "coffee-script": "~1.12.2",
+    "coffeescript": "~1.12.2",
```

Comment:
I see you rename this here but I don't see the name updated anywhere else?  That seems like a mistake?

Reply:
That's was actually for balena-config-karma to work.